### PR TITLE
Add Mission descriptions for Istio

### PR DESCRIPTION
**Pull request by kenfinnigan**

### CODE REVIEW

**Comment by kenfinnigan** [comment]
@cmoulliard @rhoads-zach either of you have concerns with the wording/naming I've added for the Istio Mission definitions?

**Comment by metacosm** [comment]
FYI, we had added similar descriptions earlier today in our launcher catalog fork along with booster description as well, so we'd appreciate having a second look from @inoxx03 as well… Descriptions can be found in the `istio` branch of our fork: https://github.com/snowdrop/launcher-booster-catalog/tree/istio

File: metadata.yaml [hunk 1]
```diff
@@ -24,6 +24,26 @@ missions:
   description: The Health Check mission demonstrates the health check pattern. When you deploy an application, its important to know if it is available and if it can start handling incoming requests. Implementing the health check pattern allows you to monitor the health of an application, which includes if an application is available and whether it is able to service requests.
   metadata:
     level: foundational
+- id: istio-circuit-breaker
```
Comment (cmoulliard) [inline metadata.yaml]: As we will use the F8 Launcher filter option, then this field should be added for every istio mission under `metadata` -> https://github.com/snowdrop/launcher-booster-catalog/blob/istio/metadata.yaml#L44
@kenfinnigan Can you take care of that ?

File: metadata.yaml [hunk 2]
```diff
@@ -24,6 +24,26 @@ missions:
...
+  name: Istio - Circuit Breaker
+  description: The Istio Circuit Breaker mission demonstrates limiting access to a degraded service until it becomes available to handle requests. This helps prevent cascading failure in other services that depend on the failed services for functionality.
```
Comment (rhoads-zach) [inline metadata.yaml]: >  This helps prevent cascading failure in other services that

to 

>  This helps prevent cascading failures in other services that

**Comment by metacosm** [comment]
What happens on the UI if you have missions without associated boosters?

**Comment by kenfinnigan** [comment]
The DevExp said to me earlier this week that nothing is shown.

I will be updating the PR this morning

File: metadata.yaml [hunk 3]
```diff
@@ -24,6 +24,26 @@ missions:
...
+  metadata:
+    level: expert
+- id: istio-distributed-tracing
+  name: Istio - Distributed Tracing
+  description: The Istio Distributed Tracing mission demonstrates how to integrate tracing into services running on Istio.
+  metadata:
+    level: expert
+- id: istio-mutual-tls-acl
+  name: Istio - Mutual TLS and ACL
```
Comment (inoxx03) [inline metadata.yaml]: FYI: According to the recent discussion on Slack, the Spring Boot team are considering renaming this mission to _Istio Security_.

**Comment by gastaldi** [comment]
@kenfinnigan yes, that's correct, if there are no boosters using the mission, it doesn't make sense to show that

**Comment by kenfinnigan** [comment]
All, I've just pushed the updates that have been requested.

Please review and let me know if there's more changes needed

**Comment by rhoads-zach** [comment]
sorry for the delay. approved

**Comment by kenfinnigan** [comment]
@gastaldi is CI dead? Didn't think jobs take this long?

**Comment by gastaldi** [comment]
[test]

**Comment by gastaldi** [comment]
CI was broken yesterday, let's see what happens now

**Comment by gastaldi** [comment]
Ok, all good. Merging this

**Comment by metacosm** [comment]
Thanks!

**Pull request merged**